.github/workflows/ci-nodejs.yml

```diff
@@ -433,7 +431,7 @@ jobs:
       - name: Test CLI
         if: ${{ env.TEST_PREVIOUSLY_PASSED != 'true' }}
         run: |
-          npm run cli:test
+          npm run test --prefix cli
```

Comment:
Insteresting, what's the difference? I see in package.json we already have:
`"cli:test": "npm run test --prefix cli",` so it should already be running that?